solidity/contracts/bridge/Bridge.sol

```diff
@@ -138,12 +197,69 @@ contract Bridge is Ownable {
     ///         execute a sweep.
     mapping(uint256 => DepositInfo) public deposits;
 
-    /// @notice Maps the wallet public key hash (computed using HASH160 opcode)
-    ///         to the latest wallet's main UTXO computed as
+    /// @notice Maps the 20-byte wallet public key hash (computed using HASH160
+    ///         opcode) to the latest wallet's main UTXO computed as
     ///         keccak256(txHash | txOutputIndex | txOutputValue). The `tx`
     ///         prefix refers to the transaction which created that main UTXO.
+    ///         The txHash is LE bytes32, txOutputIndex an uint32, and
```

Comment:
Bytes array are always read in the same order, we can talk about LE/BE when the conversion happens.

Reply:
Don't forget we are in Bitcoin land where transaction hashes are either presented as big-endian (block explorers, etc.) or little-endian (internally in the protocol). This is why I think we need to make it explicit.

Reply:
Reworded it a bit. See dbb7bf37